### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by mischah**

### CODE REVIEW

**Comment by stefanjudis** [comment]
Hmm... can't merge. :(

**Comment by mischah** [comment]
No problem. Going to resolve and force push from my side.

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 ## About
 
-Forrest is an npm desktop client to deal with daily work flows. It lets you control common npm commands and all custom scripts defined in the `package.json`.
+Forrest is an npm desktop client to deal with daily work flows. It let’s you control common npm commands and all custom scripts defined in the `package.json`.
```
Comment (stefanjudis) [inline README.md:15]: but this was actually correct. :)

http://www.dict.cc/?s=lets

**Comment by mischah** [comment]
uhm. jup. guess its too late 🙈

**Pull request closed without merging**